pkg/uefi/meregion.go

```diff
@@ -0,0 +1,228 @@
+// Copyright 2019 the LinuxBoot Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package uefi
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"log"
+)
+
+// ME Partition parsing, the goal is to spot a padding in the ME Region
+// after the ME partitions so that this region can be shrunk in the IFD.
+//
+// ME partition informations from http://me.bios.io/ME_blob_format
+
+// MEFTPSignature is the sequence of bytes that an ME Flash Partition
+// Table is expected to start with.
+var (
+	MEFTPSignature = []byte{0x24, 0x46, 0x50, 0x54}
+)
+
+const (
+	// MEFTPSignatureLength represents the size of the ME FTP signature
+	MEFTPSignatureLength = 4
```

Comment:
Instead of having this constant, can you use `len(MEFTPSignature)` throughout?

Reply:
Right but then `FlashSignatureLength` in flash.go should also be changed ;)